### PR TITLE
Rename and delete map

### PR DESCRIPTION
**Pull request by nobuti**
This PR implements map renaming and removing actions.
This belongs to #8302.

cc @xavijam

### CODE REVIEW

**Comment by saleiva** [comment]
VAMOO!

File: lib/assets/javascripts/cartodb3/editor/editor-header.js [hunk 1]
```diff
@@ -49,6 +75,119 @@ module.exports = CoreView.extend({
     this.add_related_model(this._editorModel);
   },
 
+  _renameMap: function () {
+    var newName = this._inlineEditor.getValue();
+
+    if (newName !== '') {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/editor-header.js]: And different than the previous name?

File: lib/assets/javascripts/cartodb3/editor/map-operations/remove-map.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  var name = visDefinitionModel.get('name');
+
+  visDefinitionModel.destroy({
+    wait: true,
+    success: function (mdl, attrs) {
+      successCallback && successCallback(mdl, attrs);
+    },
+    error: function (mdl, e) {
+      errorCallback && errorCallback(mdl, e);
+      Notifier.addNotification({
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/map-operations/remove-map.js]: 👍

**Comment by xavijam** [comment]
:+1: 🚢 :it:

**Comment by xavijam** [comment]
I'd add the small details I suggested about check if the name is different than the previous one, and then merge it!

**Comment by nobuti** [comment]
Frontend passing 🚀

**Pull request merged**